lib/lita/handlers/giphy.rb

```diff
@@ -2,15 +2,17 @@
 module Lita
   module Handlers
     class Giphy < Handler
+
+      config :api_key, required: true
+      config :limit, default: 10
```

Comment:
Whoops, it doesn't look like this config option is actually used anywhere. Mind removing this line?